src/main/java/com/google/songgame/servlets/RoomServlet.java

```diff
@@ -1,25 +1,59 @@
 package com.google.songgame.servlets;
 
 import com.google.gson.Gson;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.stream.Collectors;
+import java.util.Collections;
 
 import org.apache.hc.core5.http.ParseException;
 import com.google.gson.Gson; 
-import java.util.ArrayList;
+import java.util.*;
```

Comment:
Generally not a good idea to use "wildcard imports" (import.whatever.*), mentioned in the Google Java Styleguide (https://google.github.io/styleguide/javaguide.html#s3.3-import-statements). Change to only import classes you need.

Reply:
@salnad ooh i missed that

Reply:
Hi Isis, did you change this in a later commit but not push it? I still see the wildcard import. In any case, could you remove it and add only the specific imports you need?